src/main/java/mate/academy/model/User.java

```diff
@@ -0,0 +1,61 @@
+package mate.academy.model;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "users")
+public class User {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    private String email;
```

Comment:
```suggestion
    @Column(unique = true)
    private String email;
```
checklist: Let’s make the email unique. We need to check the email during user registration and make the email a unique field in the database.